environment/src/main/java/com/sequenceiq/environment/exception/ExperienceOperationFailedException.java

```diff
@@ -0,0 +1,17 @@
+package com.sequenceiq.environment.exception;
+
+public class ExperienceOperationFailedException extends RuntimeException {
```

Comment:
it could break a rest request? If yes, can you create a mapper for this?

Reply:
no, we do not throw this exception during REST request
but there is a small piece of exception handler that needs modified